app/templates/cookies/cookie_settings.html

```diff
@@ -0,0 +1,123 @@
+  </div>
+
+  <div id="cookie-settings-confirmation" class="banner-information-without-action cookie-settings__confirmation">
+    <h2 class="govuk-heading-xmedium">Your cookie settings were saved</h2>
+
+    <p class="govuk-body">You can change your preferences at any time on this page.</p>
+    <p class="govuk-body">
+        <a href="{{ url_for('external.index') }}" class="cookie-settings__prev-page">Go back to the page you were looking at.</a>
+    </p>
+
+  </div>
+
+  <div id="cookie-settings-warning" class="banner-information-without-action">
+    <h2 class="govuk-heading-xmedium">Your cookie settings have not yet been saved</h2>
+
+    <p class="govuk-body">
+        Digital Marketplace sets cookies when you visit our website.
+        You can choose to change these settings to your own preferences.
+        You need to save this page with your new choices.
+    </p>
+  </div>
+  <h1 class="govuk-heading-xl">Change your cookie settings</h1>
+
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+
+      <p class="govuk-body">
+        Cookies are files saved on your phone, tablet or computer when you visit a website.
+      </p>
+
+      <p class="govuk-body">
+        We use cookies to store information about how you use the Digital Marketplace, such as the pages you visit.
+      </p>
+
+      <h2 class="govuk-heading-l">Cookie settings</h2>
+      <div class="cookie-settings__no-js">
+        <div class="govuk-govspeak">
+          <p>We use Javascript to set most of our cookies. Unfortunately Javascript is not running on your browser, so you cannot change your settings. You can try:</p>
+          <ul>
+            <li>reloading the page</li>
+            <li>turning on Javascript in your browser</li>
+          </ul>
+        </div>
+      </div>
+
+      <div class="cookie-settings__form-wrapper">
+        <p class="govuk-body">We use two types of cookies.</p>
+
+        <form id="cookie-settings">
+          <h3 class="govuk-heading-m">Cookies that measure website use</h3>
+
+          <p class="govuk-body">
+            We use Google Analytics to measure how you use the Digital Marketplace so we can improve it based on user needs.
+            Google Analytics sets cookies that store anonymised information about:
+          </p>
+
+          <ul class="govuk-list govuk-list--bullet">
+            <li>how you got to Digital Marketplace</li>
+            <li>the pages you visit on Digital Marketplace and government digital services, and how long you spend on each page</li>
+            <li>what you click on while you&rsquo;re using the service</li>
+          </ul>
+
+          <p class="govuk-body">
+            We do not allow Google to use or share the data about how you use this site.
+          </p>
+
+          <p class="govuk-body">
+            You can choose whether you&rsquo;re happy for us to use these cookies.
```

Comment:
Please amend to: Do you want to accept analytics cookies?

Radio buttons: Yes and No

Save cookie settings
![image](https://user-images.githubusercontent.com/57447099/73525006-33eec500-4406-11ea-94aa-0e68e535397f.png)